include/zephyr/pm/policy.h

```diff
@@ -68,6 +68,7 @@ struct pm_policy_event {
 	/** @cond INTERNAL_HIDDEN */
 	sys_snode_t node;
 	uint32_t value_cyc;
+	bool low_latency;
```

Comment:
Should this be wrapped in `#ifdef CONFIG_PM_LOW_LATENCY_EVENTS`? Probably there is no reason to reserve this bit if PM_LOW_LATENCY_EVENTS is disabled.